VMBackup/main/handle.py

```diff
@@ -564,6 +565,17 @@ def enable():
         error_msg = 'Failed to call the daemon'
         exit_with_commit_log(temp_status, temp_result,error_msg, para_parser)
 
+def set_file_permission(file_path, permission):
+    os.system('chmod ' + permission + " " + file_path)
+
+def convert_dos_to_unix(file_path):
+    os.system('dos2unix ' + file_path)
```

Comment:
even dont do it all the time. check if dos2unix is needed then perform the step.